pkgs/applications/version-management/sr-ht/meta.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit
+, srht, python3Packages }:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  name = "meta-sr-ht-${version}";
+  version = "0.21.2";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~sircmpwn/meta.sr.ht";
+    rev = version;
+    sha256 = "0z45aq3i26kagpc0hy7m137w7vy8p4k0fw0ahpk5br3nizwfy043";
+  };
+
+  propagatedBuildInputs = [
+    alembic
+    bcrypt
+    # flask_login
+    pgpy
+    pystache
+    qrcode
+    redis
+    srht
+    sshpubkeys
+    stripe
+    prometheus_client
+    weasyprint
+  ];
+
+  patches = [
+    ./env.patch
+  ];
+
+  PKGVER = version;
+  SRHT_PATH = "${srht}/${python3Packages.python.sitePackages}/srht";
+
+  meta = with stdenv.lib; {
+    homepage = "https://git.sr.ht/~sircmpwn/meta.sr.ht";
+    description = "meta.sr.ht website";
```

Comment:
I'm not sure about capitalizing this one.

Reply:
I think if the description would actually describe something the problem wouldn't exist ;-)